refactor: update `blas/ext/base/scusumkbn` to follow current project conventions (#2012)
stdlib-js/stdlib: lib/node_modules/@stdlib/blas/ext/base/scusumkbn/docs/repl.txt

    Computes the cumulative sum of single-precision floating-point strided array
    elements using an improved Kahan–Babuška algorithm.

>     The `N` and `stride` parameters determine which elements in the strided

Reviewer: ```suggestion
    The `N` and stride parameters determine which elements in the strided
```

>     The `N` and stride parameters determine which elements in the strided
>     arrays accessed at runtime.

Reviewer 2: ```suggestion
    The `N` and stride parameters determine which elements in the strided arrays
    are accessed at runtime.
```

    > y0
    <Float32Array>[ 0.0, 0.0, 0.0, -2.0, 0.0, -1.0 ]

{{alias}}.ndarray( N, sum, x, strideX, offsetX, y, strideY, offsetY )
    Computes the cumulative sum of single-precision floating-point strided array
    elements using an improved Kahan–Babuška algorithm and alternative indexing
    semantics.

    While typed array views mandate a view offset based on the underlying

>     buffer, the `offsetX` and `offsetY` parameter supports indexing semantics
>     based on starting indices.

Reviewer: ```suggestion
    buffer, the offset parameters support indexing semantics based on
    starting indices.
```
